Thing.h

```diff
@@ -460,8 +458,6 @@ class ThingEventObject {
       data["data"] = *this->getValue().string;
       break;
     }
-
-    data["timestamp"] = timestamp;
```

Comment:
What's the reason for removing the timestamp?

Reply:
Well, it was never a real timestamp. So, I don't see any value in keeping it, if it only reflects the time since bootup or something random. I don't remember anymore. It was just not the real time.